filetracker/client/remote_data_store.py

```diff
@@ -131,3 +155,22 @@ def delete_file(self, filename):
             'Last-Modified': email.utils.formatdate(version)})
         # SIO-2093
         # response.raise_for_status()
+
+
+class FileLikeFromResponse(object):
```

Comment:
Since this is a private class, the name should probably start with an underscore.